llama_index/vector_stores/qdrant.py

```diff
@@ -297,13 +358,29 @@ def _create_collection(self, collection_name: str, vector_size: int) -> None:
         """Create a Qdrant collection."""
         from qdrant_client.http import models as rest
 
-        self._client.recreate_collection(
-            collection_name=collection_name,
-            vectors_config=rest.VectorParams(
-                size=vector_size,
-                distance=rest.Distance.COSINE,
-            ),
-        )
+        if self.enable_hybrid:
+            self._client.recreate_collection(
+                collection_name=collection_name,
+                vectors_config={
+                    "text-dense": rest.VectorParams(
+                        size=vector_size,
+                        distance=rest.Distance.COSINE,
```

Comment:
could we make this distance function as a parameter as well?

Reply:
eh, this is a rather opinionated default collection setup. Many parameters to tweak.

The user can always define the collection themselves before using with llama-index